example/generate-open-api-schema.ts

```diff
@@ -1,10 +1,12 @@
-import { writeFileSync } from "node:fs";
+import { writeFile } from "node:fs/promises";
 import { Documentation } from "../src";
 import { config } from "./config";
 import { routing } from "./routing";
-import manifest from "../package.json";
 
-writeFileSync(
+const manifest = (await import("../package.json", { assert: { type: "json" } }))
+  .default;
```

Comment:
in ESM it requires assertion, but this way was only one that worked.

```ts
import manifest from "../package.json" assert { type: "json" };
```

that led to 

```
TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///../package.json" needs an import assertion of type "json"
```

which is probably an issue of `@swc-node/register/esm` loader

Reply:
fixed by #1376 and restored in 5c02348c